### PR TITLE
doc: add TCK logs

### PR DESCRIPTION
**Pull request by jlordiales**
- Update README to reflect that the library fully passes the TCK spec
- Add the logs from the latest run

### CODE REVIEW

File: tck.log
Comment (serroba) [inline tck.log:1]: Is it need to commit the logs?

Comment (serroba) [reply tck.log:1]: Can we place the log in a folder? Feels odd to me. Unless this is expected

Comment (jlordiales) [reply tck.log:1]: figured it's a good way to actually show that all tck tests ran and passed. Some proof that we are not just claiming to be compliant if you will.

Happy to remove it if we don't think that's useful though

**Review by serroba — APPROVED** [review]

File: tck.log
Comment (serroba) [reply tck.log:1]: I don't mind at all. I was only thinking to have it in a dedicated place

**Pull request merged**